src/pretix/plugins/sendmail/signals.py

```diff
@@ -19,5 +20,32 @@ def control_nav_import(sender, request=None, **kwargs):
             }),
             'active': (url.namespace == 'plugins:sendmail' and url.url_name == 'send'),
             'icon': 'envelope',
-        }
+            'children': [
+                {
+                    'label': _('Send email'),
+                    'url': reverse('plugins:sendmail:send', kwargs={
+                        'event': request.event.slug,
+                        'organizer': request.event.organizer.slug,
+                    }),
+                    'active': (url.namespace == 'plugins:sendmail' and url.url_name == 'send'),
+                },
+                {
+                    'label': _('Email history'),
+                    'url': reverse('plugins:sendmail:history', kwargs={
+                        'event': request.event.slug,
+                        'organizer': request.event.organizer.slug,
+                    }),
+                    'active': (url.namespace == 'plugins:sendmail' and url.url_name == 'history'),
+                },
+            ]
+        },
     ]
+
+
+@receiver(signal=logentry_display)
+def pretixcontrol_logentry_display(sender, logentry, **kwargs):
+    plains = {
+        'pretix.plugins.sendmail.sent': _('Email was sent'),
```

Comment:
We now need to handle ``pretix.plugins.sendmail.order.email.sent`` here as well in some way, otherwise the log of an order will display "pretix.plugins.sendmail.order.email.sent" directly which does not look nice. Something like "Order received a mass email"?